docs/getting-started/backstage.md

```diff
@@ -156,6 +156,21 @@ backend.add(import('@pagerduty/backstage-plugin-backend'));
 
 ```
 
+## Configure Backstage Backend API Authorization
+
+By default, this plugin will allow any unauthenticated user to make calls using your PagerDuty API token.
+
+You should consider carefully if this is appropriate in your production environment.
+
+To disable this behaviour, you can set `pagerDuty.disableUnauthenticatedAccess` to true.
+
+Example `app-config.yaml` excerpt:
+
+```yaml
+pagerduty:
+  disableUnauthenticatedAccess: true
+```
```

Comment:
Hello @jkueh! I reached out to you on another PR related to this matter. I think we should flip the enablement logic. It should be secure by default and less secure on demand. We would release a minor version to avoid people getting dragged by a changed like that on a patch version.